commons/ava_networks/fixed_gecko_network/fixed_gecko_network.go

```diff
@@ -45,8 +45,8 @@ func NewFixedGeckoNetworkLoader(numNodes int, numBootNodes int, isStaking bool)
 	/*
 	  TODO Implement more than one bootnode for staking.
```

Comment:
Can we change this guy? TODO blindness! 😄

Reply:
done